tests/integration/sentinel/schedules/clear-failing-tasks.spec.js

```diff
@@ -0,0 +1,138 @@
+const utils = require('@utils');
```

Comment:
I think the name of this file can create confusion, because a failing task in webapp terms is a rules-engine task with a failed status. 
A safer name would be: "clear invalid sheduled_task" . 
Also, because this is just one test, please consider adding it to the due_tasks e2e test file.